typescript/packages/subsurface-viewer/src/layers/grid3d/grid3dLayer.ts

```diff
@@ -273,8 +263,8 @@ export default class Grid3DLayer extends CompositeLayer<Grid3DLayerProps> {
             !isEqual(props.pointsData, oldProps.pointsData) ||
```

Comment:
These are deep equals, all the individual values are tested :(
I think these should all be changed to === comparisons !

Reply:
Sometimes deep comparisons are needed. Like for arrays (checks element by element) But sometimes its not needed like testing booleans. However I do not think there is a performance penalty for using deep compare. But if you want I can change it for those elements that do not need deep comparison.

Reply:
But this can be done in a separate PR...

Reply:
Ok I changed it. I guess it would be very rare for the user to change individual elements in an input array as you say.